### ISSUE TITLE
Is it really supposed to be used in admin?

### ISSUE THREAD

**Issue by aurmil** [ericthehacker/magento-phpnativepasswords#8]
hello

I installed (via modman) the module mainly for the admin part
but it did not re-hash my old-fashion password (in admin_user table) after a successfull login despite config enables this behavior (yes / yes / 10)

disabled every installed modules excep this one, cleaned caches many times
using Magento CE 1.9.1.0
no logged errors

thanks

**Comment by ericthehacker** [comment]
@aurmil 
Thanks for brining this to my attention.

The module has the capability to rehash customer passwords as they are used, but does not have the same capability for admin password hashes. The system config note incorrectly stated "customer or admin", which I've corrected.

Since the ability to intercept passwords and rehash requires rewrites, I opted not to implement this for admins, since a store usually has a lot more control over admins then they do over customers -- IE, it's reasonable for a store owner to require admins to change their passwords (and thereby use the native hashing algorithm without rewrites), but it's _not_ reasonable to expect customers to do this.

I added a note to the system config field and README encouraging store owners to change admin passwords as soon as possible.
